airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -979,6 +981,15 @@ def should_validate_schema_fixture(self, inputs: BasicReadTestConfig, test_stric
         else:
             return inputs.validate_schema
 
+    @pytest.fixture(name="should_validate_stream_statuses")
+    def should_validate_stream_statuses_fixture(self, inputs: BasicReadTestConfig, connector_metadata: dict):
+        is_certified = connector_metadata.get("data", {}).get("ab_internal", {}).get("ql") >= 400
```

Comment:
nit: it seems like we have this logic spread at [multiple places in the code](https://github.com/airbytehq/airbyte/blob/b479497c502f1bc8de1c304ffb295f9b2cc0f6e2/airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py#L1198). Should we extract that to ease maintenance?

Reply:
added new fixture for certified connectors